src/hotspot/share/classfile/moduleEntry.cpp

```diff
@@ -49,7 +49,7 @@
 
 ModuleEntry* ModuleEntryTable::_javabase_module = nullptr;
 
-oop ModuleEntry::module() const { return _module.resolve(); }
+oop ModuleEntry::module_oop() const { return _module_handle.resolve(); }
```

Comment:
(Pre-existing) Just wondering why this one-liner function doesn't reside in the moduleEntry.hpp?

Reply:
I think it's because `OopHandle::resolve()` is an inline function, but we are not allowed to include `xxx.inline.hpp` in non-inlined `.hpp` files.